app/views/warehouse_reports/health/eligibility_results/_aco_changes.haml

```diff
@@ -0,0 +1,19 @@
+%table.table.table-sm
+  %thead
+    %tr
+      %th Name
+      %th DOB
+      %th ACO Current?
+      %th Reported ACO Name
+  %tbody
+    - @aco_changes.each do |patient|
+      %tr
+        %td= link_to "#{patient.first_name} #{patient.last_name}", window_client_health_patient_index_path(client_id: patient.client_id)
+        %td= patient.birthdate
+        %td
+          - if aco_current?(patient)
```

Comment:
There's a helper method for this `checkmark_or_x(aco_current?(patient)` should do what you are looking for